### PR TITLE
Unify strings formatting

### PR DESCRIPTION
**Pull request by aznd**
This PR should fix inconsistency in a lot of strings. Wherever possible, f-strings are used.
However, in all tr functions, we need to use format(), since f-strings don't work there.
Don't merge yet, since I'm gonna push more commits, since I don't want to have everything in one commit.

### CODE REVIEW

File: rare/components/tabs/games/game_info/game_settings.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def find_proton_wrappers():
                 if os.path.exists(proton) and (
                     os.path.exists(compatibilitytool) or os.path.exists(toolmanifest)
                 ):
-                    wrapper = '"' + proton + '" run'
+                    wrapper = f'"{proton}" run'
```
Comment (aznd) [inline rare/components/tabs/games/game_info/game_settings.py:44]: I'm not sure I got those quotes right, since there are a lot of them...

Comment (MultisampledNight) [reply rare/components/tabs/games/game_info/game_settings.py:44]: Seems fine to me.

Comment (aznd) [reply rare/components/tabs/games/game_info/game_settings.py:44]: Good, thanks!

File: rare/components/tabs/account/__init__.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ def __init__(self):
         if not username:
             username = "Offline"
 
-        self.layout.addWidget(QLabel(self.tr("Logged in as ") + str(username)))
+        self.layout.addWidget(QLabel(self.tr("Logged in as {}").format(str(username))))
```
Comment (MultisampledNight) [inline rare/components/tabs/account/__init__.py:19]: Didn't you say `str` was not required with `str.format` on Discord?

**Review by MultisampledNight — APPROVED** [review]
LGTM, noticed one inconsistency but that's not a dealbreaker.

**Pull request merged**